netlogo-headless/src/main/mirror/Mirrorables.scala

```diff
@@ -13,34 +13,38 @@ import org.nlogo.core.AgentVariables.implicitTurtleVariables
 object Mirrorables {
 
   case object Patch extends Kind {
-    object Variables extends Enumeration {
+    object PatchVariables extends Enumeration {
```

Comment:
Well, I changed it back and it worked....not sure what was going on before - maybe I hadn't cleaned the project thoroughly enough.